ui/helpers/taskcluster.js

```diff
@@ -136,4 +137,56 @@ export const getAction = (actionArray, actionName) => {
   return action;
 };
 
+// Bug 1638424 - Transform WPT test paths to look like paths
```

Comment:
This is just a straight copy from `Push.jsx`

Reply:
The `taskcluster` function is all very specific to taskcluster login/authentication so I'm not sure this is the best place for these test path/fetch helpers. Maybe it'd be better to rename this so its clearer (`taskclusterAuth`) and put your changes in another file or in here as stand-alone helpers.